pkg/tbtc/signing.go

```diff
@@ -57,8 +50,6 @@ func newSigningRetryLoop(
 		attemptCounter:          0,
 		attemptStartBlock:       initialStartBlock,
 		attemptSeed:             attemptSeed,
-		randomRetryCounter:      0,
-		randomRetrySeed:         randomRetrySeed,
```

Comment:
I'd love to do the same change in `pkg/tbtc/dkg.go`. We still have there `attemptCounter` and `randomRetryCounter`. This or separate PR?

Reply:
DKG is a separate story. We must have distinct counters as 1 attempt does not equal 1 retry algorithm run. We start with eliminating inactive members in the 5 first attempts and then use the retry algorithm.

Reply:
Makes sense! Can we just add this explanation here:
https://github.com/keep-network/keep-core/blob/7b5a913e8f25c3ededf79c1bf2753a1b6af61a21/pkg/tbtc/dkg.go#L26-L30

Reply:
See 6c40e767